public/Day 70/script.js

```
@@ -53,6 +62,22 @@ function initializeChart() {
     const datasetKey = document.getElementById('dataset').value;
     const scheme = document.getElementById('color-scheme').value;
 
+
+    const sort = document.getElementById("sort-data")?.value;
+
+    if (sort !== "none") {
+    const combined = labels.map((l, i) => ({ l, v: data[i] }));
```

Biome (2.4.4)

[error] This variable is used before its declaration.

(lint/correctness/noInvalidUseBeforeDeclaration)